PypeS/pypepad.py

```diff
@@ -276,7 +276,7 @@ def GetSuggestionsList(self,word):
             scriptindex = self.text_input.search('vmtk',self.wordIndex[0],backwards=1)
             moduleName  = self.text_input.get( scriptindex,scriptindex+' wordend' )
             try:
-                exec('import '+moduleName)
+                exec('from . import '+moduleName)
                 exec('scriptObjectClassName =  '+moduleName+'.'+moduleName)
```

Comment:
OK. I'm going to try something like: `assert( issubclass(scriptObjectClassName, pypes.pypeScript)) ` and see how that works - it should solve the problem, but i can't help but feel like this is a wierd workaround. 

Out of curiosity, are you primarily a Java programmer? This library has some distinctive java-esque influence on it. It's not a bad thing, but it's been a bit of a challenge for me to adapt my coding style to understand it :+1: